handlers/graph.go

```diff
@@ -217,13 +218,47 @@ func buildNamespaceTrafficMap(namespace string, o options.Options, client *prome
 
 		// fetch the internally originating request traffic time-series
 		intIstioVector := promQuery(query, time.Unix(o.QueryTime, 0), client.API())
-		populateTrafficMap(trafficMap, &intIstioVector, o)
+		populateTrafficMapHttp(trafficMap, &intIstioVector, o)
 	}
 
+	// Section for TCP services
+	tcpMetric := "istio_tcp_sent_bytes_total"
+
+	// 1) query for traffic originating from "unknown" (i.e. the internet)
+	groupByTcp := "source_workload_namespace,source_workload,source_app,source_version,destination_workload_namespace,destination_service_name,destination_workload,destination_app,destination_version"
+	query = fmt.Sprintf("sum(rate(%v{reporter=\"destination\",source_workload=\"unknown\",destination_workload_namespace=\"%v\"} [%vs])) by (%v)",
```

Comment:
It's a little odd to use destination_workload_namespace with unknown and destination_service_namespace below.  It would be nice to consolidate or add a comment as to why we're doing it this way.  But not a blocker.